CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# v1.35.0
+### Notable Changes
+* Add LegacyXFS parameter to support Linux <= v5.4 ([#2121](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2121),[@AndrewSirenko](https://github.com/AndrewSirenko))
```

Comment:
I would change to match what is in docs/options.md, or at-least reference that documentation. Something like:

Add `legacy-xfs` driver option. Warning: This option will be removed in a future release. It is a temporary workaround for users unable to immediately migrate off of older kernel versions. Formats XFS volumes with `bigtime=0,inobtcount=0,reflink=0`, so that they can be mounted onto nodes with linux kernel ≤ v5.4. Volumes formatted with this option may experience issues after 2038, and will be unable to use some XFS features (for example, reflinks).

OR 

Add `legacy-xfs` driver option for clusters that mount XFS volumes to nodes with linux kernel <= 5.4. Warning: This is a temporary workaround for customers unable to immediately upgrade their nodes. It will be removed in a future release. See our [options documentation](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/release-1.35/docs/options.md) for more details.

---

Happy to hear other thoughts.

Reply:
Im definitely onboard with this

Reply:
Second one is basically perfect imo, only nitpick is switch "our options" to "the options"

```
Add `legacy-xfs` driver option for clusters that mount XFS volumes to nodes with linux kernel <= 5.4. Warning: This is a temporary workaround for customers unable to immediately upgrade their nodes. It will be removed in a future release. See [the options documentation](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/release-1.35/docs/options.md) for more details.
```

Reply:
Went with second options with Connors suggested change thank you!